<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>531</width>
    <height>297</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="fabprint.qrc">
    <normaloff>:/media/FabPrint.png</normaloff>:/media/FabPrint.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="mainVLayout">
      <item>
       <layout class="QHBoxLayout" name="stateLayout">
        <property name="spacing">
         <number>12</number>
        </property>
        <property name="sizeConstraint">
         <enum>QLayout::SetFixedSize</enum>
        </property>
        <property name="topMargin">
         <number>4</number>
        </property>
        <property name="rightMargin">
         <number>3</number>
        </property>
        <property name="bottomMargin">
         <number>3</number>
        </property>
        <item>
         <widget class="QPushButton" name="connectButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>0</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>Connect</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="jobButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>34</width>
            <height>23</height>
           </size>
          </property>
          <property name="text">
           <string>Job</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="materialsButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>62</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Materials</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="printButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>41</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Print</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="topVSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>4</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QStackedWidget" name="stackedWidget">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="page">
         <widget class="QPushButton" name="pushButton">
          <property name="geometry">
           <rect>
            <x>210</x>
            <y>70</y>
            <width>75</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>PushButton</string>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="page_2">
         <widget class="QRadioButton" name="radioButton">
          <property name="geometry">
           <rect>
            <x>230</x>
            <y>80</y>
            <width>82</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>RadioButton</string>
          </property>
         </widget>
        </widget>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="fowardBackLayout">
        <item>
         <spacer name="leftFbHSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="backButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>&lt; Back</string>
          </property>
          <property name="shortcut">
           <string>Ctrl+Left</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="imdFbHSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Preferred</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="forwardButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Forward &gt;</string>
          </property>
          <property name="shortcut">
           <string>Ctrl+Right</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="rightFbHSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>531</width>
     <height>21</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="toolTip">
    <string>testing</string>
   </property>
   <property name="defaultUp">
    <bool>false</bool>
   </property>
   <property name="nativeMenuBar">
    <bool>true</bool>
   </property>
   <widget class="QMenu" name="menuOptions">
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::ActionsContextMenu</enum>
    </property>
    <property name="tearOffEnabled">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>Options</string>
    </property>
    <property name="separatorsCollapsible">
     <bool>true</bool>
    </property>
    <addaction name="actionChange_Directory"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout_FabPrint"/>
   </widget>
   <addaction name="menuOptions"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionBlah">
   <property name="text">
    <string>blah</string>
   </property>
  </action>
  <action name="actionChange_Directory">
   <property name="text">
    <string>Change Directory</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::WindowShortcut</enum>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionAbout_FabPrint">
   <property name="text">
    <string>About SeraphPrint</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionVisit_FabAtHome_org">
   <property name="text">
    <string>Visit SeraphRobotics.com</string>
   </property>
   <property name="shortcut">
    <string>V</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="fabprint.qrc"/>
 </resources>
 <connections/>
</ui>
